doc/api/buffer.md

```diff
@@ -194,8 +193,8 @@ Modern Web browsers follow the [WHATWG Encoding Standard][] which aliases
 both `'latin1'` and `'ISO-8859-1'` to `'win-1252'`. This means that while doing
 something like `http.get()`, if the returned charset is one of those listed in
 the WHATWG specification it is possible that the server actually returned
-win-1252-encoded data, and using `'latin1'` encoding may incorrectly decode the
-characters.
+`'win-1252'`-encoded data, and using `'latin1'` encoding may incorrectly decode
```

Comment:
Non-blocking nit, but I don't think _win-1252_ should be in quotes or in backticks. It is not an argument to a function or a value or code. It's just text.